src/pybind/mgr/tox.ini

```diff
@@ -180,7 +180,7 @@ modules = smb
 [testenv:check-black]
 deps = {[black]deps}
 commands =
-    black --check -q {[black]options} {[black]modules}
+    black -v --check -q {[black]options} {[black]modules}
```

Comment:
Will it be a good idea to keep `-v` permanently? It will help diagnosing future failures.

Reply:
It's a little noisy, and it didn't help debug this failure as it was not caused by black, but I can consider it for future needs.